dev/jsx/weather.jsx

```diff
@@ -23,10 +23,10 @@ class FetchWeather extends React.Component {
   getWeatherData() {
     // Request data from multiple APIs simultaneously
     axios.all([
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/conditions/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/forecast/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/hourly/q/${this.props.params.state}/${this.props.params.city}.json`),
-      axios.get(`http://api.wunderground.com/api/5332856fca0fe1e7/forecast10day/q/${this.props.params.state}/${this.props.params.city}.json`)
+      axios.get(`https://api.wunderground.com/api/5332856fca0fe1e7/conditions/q/${this.props.params.state}/${this.props.params.city}.json`),
```

Comment:
You should make this a variable: 
```
const API_URL  = 'https://api.wunderground.com/api/5332856fca0fe1e7/'; 
```
then use it like so in your axios.get requests: 
```
 axios.get(`${API_URL}conditions/q/${this.props.params.state}/${this.props.params.city}.json`)
```